### DISCUSSION TITLE
Tự học lập trình

### DISCUSSION THREAD

**Discussion by huypham0808** [OngDev/community#236] [Q&A]
Hi Ông  dev. Mình có một phân vân là khi tự học lập trình thì mình nên học theo trình tự, học hết các bài giảng về C++ sau đó mới qua học cái khác như là Java, C#, các thứ về Front-end. Mình có nên học xen kẽ không: ví dụ : 2-4-6 học C++, 3-5-7 học về Java... Nhờ Ông dev cho lời khuyên.

**Comment by kics223w1** [comment]
Học Javascript sau khi học xong C++ là đẹp bạn. Javascript, React, TypeScript kiểu gì bạn cũng đụng lúc làm Front-end à.
